ee/desktop/user/menu/action.go

```diff
@@ -51,6 +52,12 @@ func (a *Action) UnmarshalJSON(data []byte) error {
 		a.Performer = openURL
 	case Flare:
 		a.Performer = actionFlare{}
+	case Message:
```

Comment:
I think `Message` is a little vague -- I could see it being confusing down the line. Maybe `MessageX` to indicate the recipient of the message -- like`MessageK2`, `MessageControl`, `MessageCloud`?

Reply:
updated to MessageControl